src/SFA.DAS.EmployerAccounts.MessageHandlers/EventHandlers/AccountUserReadStore/AccountUserRemovedEventHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using SFA.DAS.EmployerAccounts.Messages.Events;
+using SFA.DAS.EmployerAccounts.ReadStore.Mediator;
+using SFA.DAS.EmployerAccounts.ReadStore.Application.Commands;
+
+namespace SFA.DAS.EmployerAccounts.MessageHandlers.EventHandlers.AccountUserReadStore
```

Comment:
What do you think to renaming the namespace from `AccountUserReadStore` to `EmployerAccounts`?

Reply:
Not sure about that. I'm not sure that convenes the intent of these event handlers. I did consider ReadStore, but they are going to have multiple read stores so I settled on the longer name. EmployerAccounts is already part of our namespace.